src/auto-evo/simulation/food_source/IFoodSource.cs

```diff
@@ -0,0 +1,12 @@
+﻿public interface IFoodSource
+{
+    public float TotalEnergyAvailable();
+
+    /// <summary>
+    ///   Provides a fitness metric to determine population adjustments for species in a patch.
+    /// </summary>
+    /// <param name="microbe">The species to be evaluated.</param>
+    /// <returns>A float to represent score. Scores are only compared against other scores from the same FoodSource,
+    /// so different implementations do not need to worry about scale.</returns>
```

Comment:
If this starts to be multiline, then you might as well want a more structured version of this comment, such as the one used for `<summary>`, that is returning to the line and add spaces.